README.md

```diff
@@ -1,9 +1,16 @@
-## Class-ify
+# Class-ify
 
 [![codecov](https://codecov.io/gh/AY2223S1-CS2103T-T15-2/tp/branch/master/graph/badge.svg?token=LKPSW2V6NG)](https://codecov.io/gh/AY2223S1-CS2103T-T15-2/tp)
 
 ![Ui](docs/images/Ui.png)
 
+## Description
+Class-ify is a class management application build specifically for teachers to handle all administrative related matters.
```

Comment:
```suggestion
Class-ify is a class management application built specifically for teachers to handle all administrative-related matters.
```

Reply:
Applied suggested changes in latest commit